### PR TITLE
ACMW final bugs

### PR DESCRIPTION
**Pull request by Subaharish**
#152 
web-development
Competitive Coding 
Data Science 
pages are done

issue#
Fixing of Footer in ACM-W Event Page

<a href="https://gitpod.io/#https://github.com/aravindvnair99/FACE-Amrita-Bengaluru/pull/156"></a>

### CODE REVIEW

File: public/ACMW/css/events.css [hunk 1]
```diff
@@ -40,83 +31,62 @@
...
 	border: 0;
-	font-size: 11px;
+	font-size: 11px !important;
 	letter-spacing: 4px;
 	opacity: 0.9;
 }
-.navbar li a,
+.navbar li a, 
 .navbar .navbar-brand {
-	color: #d5d5d5;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

**Comment by aravindvnair99** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856&bid=17247438)

File: public/ACMW/css/events.css [hunk 2]
```diff
@@ -40,83 +31,62 @@
 .navbar {
 	font-family: Montserrat, sans-serif;
 	margin-bottom: 0;
-	background-color: #2d2d30;
+	background-color: #2d2d30 !important;
 	border: 0;
-	font-size: 11px;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 3]
```diff
@@ -40,83 +31,62 @@
...
+	color: #d5d5d5 !important;
 }
 .navbar-nav li a:hover {
-	color: #fff;
+	color: #fff !important;
 }
 .navbar-nav li.active a {
-	color: #fff;
-	background-color: #29292c;
+	color: #fff !important;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 4]
```diff
@@ -40,83 +31,62 @@
 .navbar {
 	font-family: Montserrat, sans-serif;
 	margin-bottom: 0;
-	background-color: #2d2d30;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 5]
```diff
@@ -40,83 +31,62 @@
...
-	color: #d5d5d5;
+	color: #d5d5d5 !important;
 }
 .navbar-nav li a:hover {
-	color: #fff;
+	color: #fff !important;
 }
 .navbar-nav li.active a {
-	color: #fff;
-	background-color: #29292c;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 6]
```diff
@@ -40,83 +31,62 @@
...
-}
-
-.header-logo {
-	float: left;
-	font-size: 50px;
-	padding: 20px 40px;
-}
-
-.header-logo p {
-	color: #da1f28;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

**Comment by aravindvnair99** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856&bid=17247438)

File: public/ACMW/css/events.css (hunk 5 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 4 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 2 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 3 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 6 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 1 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

**Comment by aravindvnair99** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856&bid=17247438)

File: public/ACMW/css/events.css [hunk 7]
```diff
@@ -62,61 +53,40 @@
...
-}
-
-li {
-	list-style: none;
-}
-
-.header {
-	background-color: #760c10;
-	color: #d7bb2b;
-	height: 100px;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 8]
```diff
@@ -42,16 +33,16 @@
...
 	letter-spacing: 4px;
 	opacity: 0.9;
 }
-.navbar li a,
+.navbar li a, 
 .navbar .navbar-brand {
 	color: #d5d5d5;
 }
 .navbar-nav li a:hover {
-	color: #fff;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

**Comment by Subaharish** [comment]
there is no header in event.css

**Comment by aravindvnair99** [comment]
> there is no header in event.css

@Subaharish Sorry, I didn't quite get you.

File: public/ACMW/css/events.css (hunk 7 above)
Comment (Subaharish) [reply public/ACMW/css/events.css]: there is no header class in event.css

**Comment by aravindvnair99** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856&bid=17247438)

File: public/ACMW/css/events.css [hunk 9]
```diff
@@ -62,61 +53,40 @@ body {
...
-
-li {
-	list-style: none;
-}
-
-.header {
-	background-color: #760c10;
-	color: #d7bb2b;
-	height: 100px;
+	background-color: #555 !important;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 10]
```diff
@@ -51,7 +42,7 @@ body {
 	color: #d5d5d5;
 }
 .navbar-nav li a:hover {
-	color: #fff;
+	color: #fff !important;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

**Comment by aravindvnair99** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856&bid=17247438)

File: public/ACMW/css/index.css [hunk 11]
```diff
@@ -125,10 +125,19 @@
 /*.dropdown-menu li a:hover {
 	background-color: red !important;
 } */
+footer .container{
```
Comment (aravindvnair99) [inline public/ACMW/css/index.css]: Codacy Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 12]
```diff
@@ -62,61 +53,33 @@ body {
 }
 .open .dropdown-toggle {
 	color: #fff;
-	background-color: #555;
+	background-color: #555 !important;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/members.css [hunk 13]
```diff
@@ -90,10 +90,19 @@
 	color: #fff;
 	background-color: #555 !important;
 }
+footer .container{
```
Comment (aravindvnair99) [inline public/ACMW/css/members.css]: Codacy Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 10 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 14]
```diff
@@ -40,7 +31,7 @@ body {
 .navbar {
 	font-family: Montserrat, sans-serif;
 	margin-bottom: 0;
-	background-color: #2d2d30;
+	background-color: #2d2d30 !important;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 15]
```diff
@@ -145,11 +108,19 @@ h1 span {
 	color: #1b80ce;
 	background-color: #fcfcfa;
 }
-
+footer .container{
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 14 above)
Comment (Subaharish) [reply public/ACMW/css/events.css]: "Important" has to be there, or else, the color of navbar is not changing

File: public/ACMW/css/events.css (hunk 10 above)
Comment (Subaharish) [reply public/ACMW/css/events.css]: "Important" has to be there, or else, the color of  links are not changing

File: public/ACMW/css/events.css (hunk 12 above)
Comment (Subaharish) [reply public/ACMW/css/events.css]: "Important" has to be there, or else, the color of navbar is not changing

Comment (aravindvnair99) [reply public/ACMW/css/events.css]: It can be made into another class to overcome this issue.

File: public/ACMW/css/events.css (hunk 10 above)
Comment (aravindvnair99) [reply public/ACMW/css/events.css]: It can be made into another class to overcome this issue.

File: public/ACMW/css/events.css (hunk 14 above)
Comment (aravindvnair99) [reply public/ACMW/css/events.css]: It can be made into another class to overcome this issue.

**Comment by aravindvnair99** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856&bid=17247438)

File: public/ACMW/css/events.css [hunk 16]
```diff
@@ -51,7 +42,7 @@
 	color: #d5d5d5;
 }
 .navbar-nav li a:hover {
-	color: #fff;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 17]
```diff
@@ -40,7 +31,7 @@
 .navbar {
 	font-family: Montserrat, sans-serif;
 	margin-bottom: 0;
-	background-color: #2d2d30;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css [hunk 18]
```diff
@@ -62,61 +53,33 @@
 }
 .open .dropdown-toggle {
 	color: #fff;
-	background-color: #555;
```
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

Comment (Subaharish) [reply public/ACMW/css/events.css]: here also we have to use it

File: public/ACMW/css/events.css (hunk 17 above)
Comment (Subaharish) [reply public/ACMW/css/events.css]: here also..

File: public/ACMW/css/events.css (hunk 16 above)
Comment (Subaharish) [reply public/ACMW/css/events.css]: here too

File: public/ACMW/css/events.css (hunk 18 above)
Comment (aravindvnair99) [reply public/ACMW/css/events.css]: Have a look at this: [Disallow !important](https://github.com/CSSLint/csslint/wiki/Disallow-!important)

File: public/ACMW/css/events.css (hunk 17 above)
Comment (aravindvnair99) [reply public/ACMW/css/events.css]: Have a look at this: [Disallow !important](https://github.com/CSSLint/csslint/wiki/Disallow-!important)

File: public/ACMW/css/events.css (hunk 16 above)
Comment (aravindvnair99) [reply public/ACMW/css/events.css]: Have a look at this: [Disallow !important](https://github.com/CSSLint/csslint/wiki/Disallow-!important)

**Comment by aravindvnair99** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856&bid=17247438)

File: public/ACMW/css/events.css (hunk 10 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 14 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/events.css (hunk 12 above)
Comment (aravindvnair99) [inline public/ACMW/css/events.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

**Comment by aravindvnair99** [comment]
@Subaharish Have a look at this: [Disallow !important](https://github.com/CSSLint/csslint/wiki/Disallow-!important)

File: public/ACMW/css/members.css [hunk 19]
```diff
@@ -90,10 +90,26 @@
...
 }
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+=======
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> replaced padding properties to single padding property
+=======
```
Comment (aravindvnair99) [inline public/ACMW/css/members.css]: Codacy Issue found: [Expected LBRACE at line 101, col 1.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

**Comment by aravindvnair99** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856&bid=17247438)

File: public/ACMW/css/index.css [hunk 20]
```diff
@@ -125,10 +125,25 @@
 /*.dropdown-menu li a:hover {
 	background-color: red !important;
 } */
+<<<<<<< HEAD
```
Comment (aravindvnair99) [inline public/ACMW/css/index.css]: Codacy Issue found: [Unexpected token '<' at line 128, col 1.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/index.css [hunk 21]
```diff
@@ -125,10 +125,25 @@
...
+=======
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> replaced padding properties to single padding property
+=======
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> e26a74ba8763494f3845f4433410f1a7af2ff448
```
Comment (aravindvnair99) [inline public/ACMW/css/index.css]: Codacy Issue found: [Unexpected token '>' at line 139, col 1.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/index.css [hunk 22]
```diff
@@ -125,10 +125,25 @@
 /*.dropdown-menu li a:hover {
 	background-color: red !important;
 } */
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
```
Comment (aravindvnair99) [inline public/ACMW/css/index.css]: Codacy Issue found: [Expected LBRACE at line 130, col 1.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/members.css [hunk 23]
```diff
@@ -108,7 +124,8 @@
 textarea {
 	resize: none;
 }
-.card {
+<<<<<<< HEAD
```
Comment (aravindvnair99) [inline public/ACMW/css/members.css]: Codacy Issue found: [Unknown word (CssSyntaxError)](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/members.css [hunk 24]
```diff
@@ -90,10 +90,26 @@
 	color: #fff;
 	background-color: #555 !important;
 }
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (aravindvnair99) [inline public/ACMW/css/members.css]: Codacy Issue found: [Expected LBRACE at line 94, col 1.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/members.css [hunk 25]
```diff
@@ -90,10 +90,26 @@
...
+=======
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> replaced padding properties to single padding property
+=======
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> e26a74ba8763494f3845f4433410f1a7af2ff448
```
Comment (aravindvnair99) [inline public/ACMW/css/members.css]: Codacy Issue found: [Unexpected token '>' at line 105, col 2.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

Comment (aravindvnair99) [inline public/ACMW/css/members.css]: Codacy Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/index.css [hunk 26]
```diff
@@ -125,10 +125,25 @@
...
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> replaced padding properties to single padding property
```
Comment (aravindvnair99) [inline public/ACMW/css/index.css]: Codacy Issue found: [Unexpected token '>' at line 134, col 1.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/index.css (hunk 21 above)
Comment (aravindvnair99) [inline public/ACMW/css/index.css]: Codacy Issue found: [The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

File: public/ACMW/css/members.css [hunk 27]
```diff
@@ -90,10 +90,26 @@
...
 	background-color: #555 !important;
 }
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+=======
+footer .container{
+	padding: 80px 120px 10px;
+}
+>>>>>>> replaced padding properties to single padding property
```
Comment (aravindvnair99) [inline public/ACMW/css/members.css]: Codacy Issue found: [Unexpected token '>' at line 100, col 1.](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

**Comment by aravindvnair99** [comment]
@Subaharish Why 94a913f?

**Comment by aravindvnair99** [comment]
Closing for #161

**Pull request closed without merging**